test/system/252-quadlet.bats

```diff
@@ -144,13 +144,16 @@ function remove_secret() {
 }
 
 @test "quadlet - basic" {
+    # Network=none is to work around a Pasta bug, can be removed once a patched Pasta is available.
+    # Ref https://github.com/containers/podman/pull/21563#issuecomment-1965145324
```

Comment:
This is the sort of comment where I like to see "FIXME" or "TODO", because I have a script that looks for those.

I also like to see *explicit* comments with version strings, or "> 2024-03", or some sort of date or version string offering a hint about when it would be good to check. Someone reading this in the year 2050 might wonder what "patched Pasta" means.